docs/section1/Rule1-2.md

```diff
@@ -0,0 +1,43 @@
+# Section 1 - Rule 1-2
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 1-2  
+**Rule Description:** The performance of the proposed design is calculated in accordance with Standard 90.1-2019 Appendix G, where Performance Cost Index = Proposed building performance (PBP) /Baseline building performance (BBP), where both the PBP and the BBP include all end-use load components associated with the building when calculating the Performance Cost Index (PCI).  
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE  
+**Appendix G Section:** G1.2.2  
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None
+
+**Evaluation Context:** RMD
+
+**Applicability Checks:** 
+1. Applies to projects where the proposed on site renewable energy production offsets less than or equal to 5% of the baseline building performance
```

Comment:
I am not following why this does not apply if on site renewables is greater than 5% since the calculation of PCI is not impact. Only the equation to compare it with the targetPCI is impacted.